app/mailers/admin_mailer.rb

```diff
@@ -24,6 +24,7 @@ def daily_uploads_report
     if @data_imports.any?
       date = Time.zone.yesterday.to_date
       mail to: "info@workhands.us",
+        cco: "sarah.lima@thoughtbot.com",
```

Comment:
Added my own email for validation, will remove it after this is merged and tested.